packages/shared/src/styles/custom.ts

```diff
@@ -0,0 +1,9 @@
+// this file was created to contain all custom temporary colors (outside the guideline)
+// and it is ideal to have this file to contain fewer colors as possible
+// we should eventually convert the custom colors to be part of the guideline once it is deemed to be used frequently
+
+export const feedSurveyBg =
+  'linear-gradient(180deg, rgba(255, 233, 35, 0.08) 0%, rgba(252, 83, 141, 0.08) 50%, rgba(113, 71, 237, 0.08) 100%)';
```

Comment:
Instead of this being a gradient and having that extra div, using a transparent main background works too. `color-mix(in srgb, var(--theme-background-default), transparent 8%)`

Reply:
It gives the same effect, and it's one less gradient to worry about

Reply:
Ah, yes. Will incorporate the fix on that and remove this one. Just for the comment's sake, I moved it.